http/swagger.yml

```diff
@@ -4375,6 +4375,86 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
   /packages/stacks:
+    get:
+      operationId: ListStacks
+      tags:
+        - InfluxPackages
+      summary: Grab a list of installed Influx packages
+      requestBody:
+        application/x-www-form-urlencoded:
+          schema:
+            type: object
+            properties:
+              orgID:
+                description: The organization id of the stacks
+                type: string
+              name[]:
+                description: A collection of names to filter the list by. The list is comma delimited.
+                type: array
+                items:
+                  type: string
+              stackID[]:
```

Comment:
not sure why, but open-api specs dont' follow the canonical array format for `application/x-www-form-urlencoded` forms... that is why you see the `[]` here